backends/firrtl/firrtl.cc

```diff
@@ -329,7 +348,8 @@ struct FirrtlWorker
 			log_warning("No instance for %s.%s\n", cell_type.c_str(), cell_name.c_str());
 			return;
 		}
-		wire_exprs.push_back(stringf("%s" "inst %s%s of %s", indent.c_str(), cell_name.c_str(), cell_name_comment.c_str(), instanceOf.c_str()));
+		auto cellFileinfo = getFileinfo(cell);
```

Comment:
FYI project maintainers don't like `auto` for simple types like `std::string`

Reply:
Done.